charts/newrelic-infra-operator/README.md

```diff
@@ -7,51 +7,52 @@ This chart will deploy the [New Relic Infrastructure Operator][1], which injects
+| `nodeSelector`                                                                  | Node label to use for scheduling.                                                                                                                                                                                                                                                                                                            | `{}`                                                                                                 |
+| `timeoutSeconds`                                                                | Seconds to wait for a webhook to respond. The timeout value must be between 1 and 30 seconds.                                                                                                                                                                                                                                                | `30`                                                                                                 |
+| `tolerations`                                                                   | List of node taints to tolerate (requires Kubernetes >= 1.6)                                                                                                                                                                                                                                                                                | `[]`                                                                                                 |
+| `affinity`                                                                      | Node affinity to use for scheduling.                                                                                                                                                                                                                                                                                                         | `{}`                                                                                                 |
+| `config.ignoreMutationErrors`                                                   | if true it instruments the operator to ignore injection error instead of failing.                                                                                                                                                                                                                                                            | `true`                                                                                               |
```

Comment:
I wonder if this is the right place to document the syntax of the configuration. Maybe it would be better to document it in operator docs instead?